backend/model/model.go

```diff
@@ -621,6 +619,23 @@ func (u *Workspace) BeforeCreate(tx *gorm.DB) (err error) {
 	return
 }
 
+func (s *Session) BeforeCreate(tx *gorm.DB) (err error) {
+	if s.LastUserInteractionTime.IsZero() {
+		s.LastUserInteractionTime = time.UnixMilli(0)
+	}
+	return
+}
+
+func (s *SystemConfiguration) BeforeCreate(tx *gorm.DB) (err error) {
+	if s.ErrorFilters == nil {
+		s.ErrorFilters = pq.StringArray{"ENOENT.*", "connect ECONNREFUSED.*"}
+	}
+	if s.IgnoredFiles == nil {
+		s.IgnoredFiles = pq.StringArray{".*/node_modules/.*", ".*/go/pkg/mod/.*", ".*/site-packages/.*"}
+	}
+	return
+}
+
```

Comment:
is this only a problem for existing models, or should we use `BeforeCreate` for all future defaults?

Reply:
The issue affected these defaults specifically, for example GORM was comparing `'1970-01-01'` against `1970-01-01` without quotes and thinking they were different defaults, but changing the default to remove quotes broke the default in Postgres. The other defaults seem to work ok so I figure it's something that will be fixed in GORM eventually. I think old versions of GORM didn't attempt to migrate the defaults.